docs/sqllab.rst

```diff
@@ -122,3 +122,22 @@ database configuration:
 
 Here, "version" should be the version of your Presto cluster. Support for this
 functionality was introduced in Presto 0.319.
+
+.. _ref_ctas_engine_config:
+
+Create Table As (CTAS)
+''''''''''''''''''''''
+
+You can use ``CREATE TABLE AS SELECT ...`` statements on SQLLab this feature can be toggled on
+and off at the database configuration level.
```

Comment:
This still feels like two sentences in one. Perhaps add a period between "SQLLab" and "this" or something similar?